File: 30-days-of-code/java/Day8.java
//Complete this code or write your own from scratch
import java.util.*;
import java.io.*;

class Day8{
    public static void main(String []argh){
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        HashMap<String, Integer> book = new HashMap<String, Integer>();
        for(int i = 0; i < n; i++){
            String name = in.next();
            int phone = in.nextInt();
            book.put(name, phone);
        }
        while(in.hasNext()){
            String s = in.next();
            if (book.containsKey(s)) {
                System.out.println(s + "=" + book.get(s));
            } else {
                System.out.println("Not found");
            }
        }
        in.close();
    }
}

File: 30-days-of-code/go/Day8.go
package main

import "fmt"

func main() {
	var n int
	fmt.Scanf("%d", &n)
	phoneBook := make(map[string]string)
	for i := 0; i < n; i++ {
		var (
			name  string
			phone string
		)
		fmt.Scan(&name)
		fmt.Scan(&phone)
		phoneBook[name] = phone
	}
	for i := 0; i < n; i++ {
		var name string
		fmt.Scan(&name)
		if phone, exists := phoneBook[name]; exists {
			fmt.Printf("%s=%s\n", name, phone)
		} else {
			fmt.Println("Not found")
		}
	}

}
